abel/direct.py

```diff
@@ -73,7 +74,7 @@ def _construct_r_grid(n, dr=None, r=None):
 
 def direct_transform(fr, dr=None, r=None, direction='inverse',
                      derivative=gradient,
-                     int_func=simpson_rule_wrong,
+                     int_func=np.trapz,
```

Comment:
Perhaps, the change to `int_func` should occur at the source level?
`def _pyabel_direct_integral(f, r, correction, int_func=simpson_rule_wrong):` and delete the redundant `simpson_rule_wrong()`. This code may be placed in the `PyAbelLegacy` repo

Reply:
I think that it's fine to delete the simpson_rule_wrong function. `scipy.integrate.simps` can also handle both even and odd sizes, so I'm unsure exactly why this function is needed.

Reply:
I agree that it's better to use the trapz/simps functions from scipy if possible.